Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -96,6 +97,8 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     using InjectedContentVector = WebExtension::InjectedContentVector;
 
     using EventListenterTypeCountedSet = HashCountedSet<WebExtensionEventListenerType, WTF::IntHash<WebKit::WebExtensionEventListenerType>, WTF::StrongEnumHashTraits<WebKit::WebExtensionEventListenerType>>;
+    using EventListenerTypeSet = HashSet<WebExtensionEventListenerType, WTF::IntHash<WebKit::WebExtensionEventListenerType>, WTF::StrongEnumHashTraits<WebKit::WebExtensionEventListenerType>>;
+    using ActionsToPerformVector = Vector<CompletionHandler<void()>>;
```

Comment:
`VoidCompletionHandlerVector` would be a better name, so it can be reused if needed.